packages/container/tests/container_test.js

```diff
@@ -561,7 +581,8 @@ QUnit.test('A deprecated `container` property is appended to every object instan
   }, 'Using the injected `container` is deprecated. Please use the `getOwner` helper instead to access the owner of this object.');
 });
 
-QUnit.test('A deprecated `container` property is appended to every object instantiated from a non-extendable factory, and a fake container is available during instantiation.', function() {
+// This is testing that container was passed as an option
+QUnit.skip('A deprecated `container` property is appended to every object instantiated from a non-extendable factory, and a fake container is available during instantiation.', function() {
```

Comment:
This test shows why we need to keep the "double extend" the default path for `lookup` until at least 2.13.